restlight-test/restlight-test-starter/src/main/java/io/esastack/restlight/test/bootstrap/FullyMockMvcBuilder.java

```diff
@@ -33,8 +34,11 @@
     public MockMvc build() {
         Restlight4SpringTest server = Restlight4SpringTest.forServer(ctx);
         server.start();
-
-        return new DefaultMockMvc(server.deployments().handler());
+        RestlightServer unWrap = server.unWrap();
+        if (unWrap instanceof FakeServer) {
+            return new DefaultMockMvc(((FakeServer) unWrap).handler);
```

Comment:
Is it necessary to validate the type of `RestlightServer`?

Reply:
Unnecessary validation deleted